src/hooks/useAsyncStorage.ts

```diff
@@ -1,17 +1,15 @@
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import { useQuery } from 'react-query';
 
-export function useAsyncStorage() {
-  const useGetItem = (key: string) =>
-    useQuery<string | null>('async-storage-get', () =>
-      AsyncStorage.getItem(key),
-    );
+export function useAsyncStorage(key: string) {
+  const itemResult = useQuery<string | null>('async-storage-get', () =>
+    AsyncStorage.getItem(key),
+  );
 
-  const setItem = (key: string, value: string) =>
-    AsyncStorage.setItem(key, value);
+  const setItem = (value: string) =>
+    AsyncStorage.setItem(key, value).catch((error) =>
+      console.error(`[LifeOmicSDK]:${error}`),
```

Comment:
I think in a future PR I will purpose a wrapper or something around console to include that prefix.